lib/openstudio-standards/standards/ashrae_90_1_prm/docs/set_baseline_wwr.md

```diff
@@ -0,0 +1,191 @@
+Set window to wall ratio in the baseline model
+
+# ASHRAE 90.1-2019
+- G3.1 (5) Baseline c: Vertical Fenestration Areas: For building area types included in the Table G3.1.1-1, vertical fenestration areas for new buildings and additions shall equal that in Table G3.1.1-1 based on the area of gross above-grade walls that separate conditioned spaces and semiheated spaces from the exterior. 
+- G3.1 (5) Baseline c: Where a building has multiple building area types, each type shall use the values in the table. 
+- G3.1 (5) Baseline c: The vertical fenestration shall be distributed on each face of the building in the same proportion as in the proposed design. 
+- G3.1 (5) Baseline c: For building areas not shown in Table G3.1.1-1, vertical fenestration areas for new building and additions shall equal that in the proposed design or 40% of gross above-grade wall area, whichever is smaller, and shall be distributed on each face of the building in the same proportions in the proposed design.
+- G3.1 (5) Baseline c: ([addendum_l](https://www.ashrae.org/file%20library/technical%20resources/standards%20and%20guidelines/standards%20addenda/90_1_2019_l_20201030.pdf)): If this would cause the combined vertical fenestration and opaque door area on a given face to exceed the gross above-grade wall area on that face, then the vertical fenestration area on other faces shall be increased in proportion to the gross above-grade wall area of these faces such that the total baseline building vertical fenestration area is equal to that calculated following Table G3.1.1.-1.
+- G3.1 (5) Baseline c: The fenestration area for an existing building shall equal the existing fenestration area prior to the proposed work and shall be distributed on each face of the building in the same proportions as the existing building.
+
+# Code Requirement Interpretation
+1. Fenestration area shall be calculated based on the area of gross above-grade walls that separate conditioned spaces and semiheated spaces from the exterior
+2. If the window to wall ratio building type (wwr_building_type) can be found in Table G3.1.1-1, then the baseline window to wall ratio shall equal to the values specified in the table by proportionally decrease or increase the fenestration area.
+3. If the wwr_building_type cannot be found in the Table or it is other type, the baseline window to wall ratio shall equal to 40% or proposed window to wall ratio, whichever is smaller.
+4. Multiple wwr_building_types in a building shall use the values in the table correspondingly.
+5. If the fenestration area increase exceeds the host wall surfaces, then the residual shall be distributed to the other wall surfaces proportionally to its wall surface area
+6. Existing building shall remain the fenestration area the same as the existing fenestration area prior to the proposed work
```

Comment:
Are we handling existing building using the user data? I may have missed it but I have not seen anything related to handling that feature for now.

Reply:
I think I missed the implementation of this part... I will add it to the code in the next push

Reply:
A follow up questions @lymereJ @dmaddoxwhite do you have preference on where to add this check?
I can do it at high-level such as somewhere at the beginning of the [function](https://github.com/NREL/openstudio-standards/blob/AppendixG_Dev/lib/openstudio-standards/standards/Standards.Model.rb#L4450-L4454)
- Pro: avoid all the calculations and API calls - save time especially for large models.
- Con: we might need to create a dummy function again in Standard.Model.rb and override it in the PRM routine.

I can also do it at sub [function](https://github.com/NREL/openstudio-standards/blob/wwr_increase_feature/lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.Model.rb#L2033-L2043) level.
- Pro: no structure change to parent class.
- Con: Will still require all the calculation and API calls.

Reply:
Mmmh, good one. I think I would lean towards the first solution you proposed, mostly for the pro, but I don't have a strong opinion on it.

Reply:
I agree with first solution.

Reply:
As discussed, we will keep the existing building feature for the next release.